models/project/board.go

```diff
@@ -157,30 +156,30 @@ func deleteBoardByID(ctx context.Context, boardID int64) error {
 		return err
 	}
 
-	if err = board.removeIssues(e); err != nil {
+	if err = board.removeIssues(ctx); err != nil {
 		return err
 	}
 
-	if _, err := e.ID(board.ID).Delete(board); err != nil {
+	if _, err := db.GetEngine(ctx).ID(board.ID).NoAutoCondition().Delete(board); err != nil {
```

Comment:
Why is this needed?

Reply:
`NoAutoCondition` means only use `ID` become the delete condition but not read any field from `board` here. It has performance improvement.